lib/engine/game/base.rb

```diff
@@ -2041,7 +2041,10 @@ def game_ending_description
 
       def action_processed(_action)
         @corporations.dup.each do |corporation|
-          close_corporation(corporation) if corporation.share_price&.type == :close
+          if corporation.share_price&.type == :close
+            close_corporation(corporation)
+            corporation.close!
```

Comment:
calling `corporation.close!` inside of `close_corporation()` instead of here breaks a few 1861/1867 games at the merger step. I'm not familiar with those games yet so I didn't dig deeper to see if this should be fixed another way.

1849 and 18CO both also have code that calls `close_corporation(corporation)` and then separately calls `corporation.close!`.